src/main/python/api.py

```diff
@@ -6,7 +6,13 @@
 from flask import Flask, jsonify, request
 # FIXME: separate this out to a classifier class where we can switch out the models
 from pyserini import Pyserini
-from jaccard import Jaccard
+from sm_model.bridge import SMModelBridge
```

Comment:
How does this work? I get ModuleNotFoundError: No module named 'sm_model'? sm_model is a part of Castor.